config/exp/etcdrestore/default/manager_role_aggregation_patch.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: manager-role
+  labels:
+    turtles-capi.cattle.io/aggregate-to-manager: "true"
```

Comment:
Does this label match the label used in the aggregated ClusterRole ?

Reply:
Good catch @yiannistri, pushed https://github.com/rancher/turtles/pull/506/commits/952ff064abf6757c008153e7460abf06239ff3c7